content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -4,4 +4,39 @@ description: Istio 1.4 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Added** support for fractional [request mirroring](/docs/tasks/traffic-management/mirroring/).
+- **Improved** the Envoy sidecar to exit upon crashing, giving better visibility of health.
```

Comment:
Not really.. previously it was completely obscured, now its just standard kubernetes. if it gets OOM killed kubernetes will mark it as OOMKilled. If it is crashing it will be marked as CrashLoopBackoff. Its both self explanatory and part of basic kubernetes docs, so I don't think we need to specifically document it on our tasks